libsmartcols/src/Makemodule.am

```diff
@@ -21,7 +20,30 @@ libsmartcols_la_SOURCES= \
 	libsmartcols/src/calculate.c \
 	libsmartcols/src/grouping.c \
 	libsmartcols/src/walk.c \
-	libsmartcols/src/init.c
+	libsmartcols/src/init.c \
+	\
+	libsmartcols/src/filter-parser.c \
+	libsmartcols/src/filter-scanner.c \
+	libsmartcols/src/filter-parser.h \
+	libsmartcols/src/filter-scanner.h \
+	\
+	libsmartcols/src/filter.c \
+	libsmartcols/src/filter-param.c \
+	libsmartcols/src/filter-expr.c
+
+BUILT_SOURCES += libsmartcols/src/filter-parser.c \
+		 libsmartcols/src/filter-parser.h \
+		 libsmartcols/src/filter-scanner.c \
+		 libsmartcols/src/filter-scanner.h
+
+EXTRA_DIST += libsmartcols/src/filter-parser.y \
+	      libsmartcols/src/filter-scanner.l
+
+libsmartcols/src/filter-parser.h libsmartcols/src/filter-parser.c: libsmartcols/src/filter-parser.y libsmartcols/src/filter-scanner.h
```

Comment:
Technically this runs bison twice.
See https://www.gnu.org/software/automake/manual/html_node/Multiple-Outputs.html

Reply:
Good point. I'll try to use .stamp file here (we already use this way in config/gtk-doc.make and po-man/Makefile.am). 

Ah, Dear autotools ...